code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -95,3 +94,249 @@
 	force = 2 //Also very weak because it's smaller
 	suppressed = 1 //Softer fire sound
 	can_unsuppress = 0 //Permanently silenced
+
+// Not quite a syringe gun, but also not completely unlike one either.
+// these names are subject to change, I just want to leave stuff in place for now.
+/obj/item/gun/syringe/rapidsyringe
+	name = "rapid syringe gun"
+	desc = "A syndicate rapid syringe gun stolen from NanoTrasen's R&D archives. Has an internal mechanism capable of storing and filling syringes from an internal reservoir.\nIt has a large flap on the side that you can dump a box or bag of syringes into, and a port for filling it with liquid."
```

Comment:
if NT cannot make them from RND, how were they stolen? or does the `archives` reference the fact that it used to exist?